script.js

```diff
@@ -1,4 +1,348 @@
+      ". Computer score = " +
+      computerScore +
+      "<br><br> Enter 'hit' to draw another card, or 'stand' to end your turn.";
+
+    currentGameMode = "hit or stand";
+
+    // If computer score < 17, computer has to draw card again
+    if (computerScore < 17) {
+      myOutputValue =
+        "You drew a " +
+        playerCards[0].name +
+        " and " +
+        playerCards[1].name +
+        ". Your score = " +
+        playerScore +
+        "<br><br>Computer drew " +
+        computerCards[0].name +
+        " and " +
+        computerCards[1].name +
+        ". Computer score = " +
+        computerScore +
+        "<br><br> Computer has to draw again. Click submit to continue.";
+
+      currentGameMode = "computer draws";
+    }
+  } else if (currentGameMode == "hit or stand") {
+    // If player chooses to stand
+    if (input == "stand") {
+      computerScore = calculateComputerScore();
+      playerScore = calculatePlayerScore();
+      myOutputValue = endGame(playerScore, computerScore);
+
+      currentGameMode = "draw card";
+      playerCards = [];
+      computerCards = [];
+    }
+    // If player chooses to hit
+    if (input == "hit") {
+      playerCards.push(playerDeck.pop());
+      playerScore = calculatePlayerScore();
+      computerScore = calculateComputerScore();
+
+      var newPlayerIndex = playerCards.length - 1;
+
+      myOutputValue =
+        "You drew " +
+        playerCards[newPlayerIndex].name +
+        ". Your score = " +
+        playerScore +
+        ". Computer score = " +
+        computerScore +
+        "<br><br> Enter 'hit' to draw another card, or 'stand' to end your turn.";
+
+      console.log("player score: " + playerScore);
+      console.log("computer score" + computerScore);
+
+      // If player's new score exceeds 21, game automatically ends.
+      if (playerScore > 21) {
+        myOutputValue =
+          "You drew " +
+          playerCards[newPlayerIndex].name +
+          ". Your score = " +
+          playerScore +
+          ". Computer score = " +
+          computerScore +
+          "<br><br> Yikes! You lost, your score exceeded 21. Better luck next time! <br><br> Click submit to play again.";
+        currentGameMode = "draw card";
+        playerCards = [];
+        computerCards = [];
+      }
+    }
+  } else if ((currentGameMode = "computer draws")) {
+    computerCards.push(computerDeck.pop());
+    computerScore = calculateComputerScore();
+    console.log("computer score: " + computerScore);
+    playerScore = calculatePlayerScore();
+
+    var newComputerIndex = computerCards.length - 1;
+
+    if (computerScore < 17) {
+      myOutputValue =
+        "Computer drew " +
+        computerCards[newComputerIndex].name +
+        ". Your score = " +
+        playerScore +
+        ". Computer score = " +
+        computerScore +
+        ". <br><br> Computer has to draw again. Click submit to continue.";
+
+      currentGameMode = "computer draws";
```

Comment:
This line may not be necessary, because the `currentGameMode` would have to already be `"computer draws"` for this line of logic to be executed. You can leave this out, let the program reach the `return` on [line 288](https://github.com/rocketacademy/basics-blackjack/blob/a122772c1c05a29aae43234d1caabdbe74894d05/script.js#L288) to print the string to the interface. Subsequent clicks of Submit may still allow the program to stay in the `"computer draws"` game mode until it finally has a `computerScore` of 17 and above.